auth.go

```diff
@@ -472,10 +472,11 @@ func (x *Central) Login(username, password string) (sessionkey string, token *To
 		Identity: user.getIdentity(),
 		UserId:   user.UserId,
 	}
+
 	sessionExpiry := time.Now().Add(x.NewSessionExpiresAfter)
-	if x.PasswordExpiresAfter != 0 {
+	if x.PasswordExpiresAfter != 0 && x.ldap == nil {
```

Comment:
@bmharper can we, for the moment, just check whether it is an ldap call and skip password expiry check?

Reply:
I definitely wouldn't say "for the moment". Any code that gets committed is likely to remain there forever. But will this approach work for a hybrid of ldap and IMQS users in the system? This is actually quite a common scenario.